src/pages/General/experimentalFeatures/ExperimentalFeatures.js

```diff
@@ -0,0 +1,69 @@
+import i18n from '@dhis2/d2-i18n'
+import { Legend } from '@dhis2/ui'
+import isEmpty from 'lodash/isEmpty'
+import PropTypes from 'prop-types'
+import React, { useEffect, useState } from 'react'
+import { CheckboxField } from '../../../components/field'
+import { MoreOptions } from '../../../components/moreOptions'
+import { getValidKeysList } from '../helper'
+import styles from './ExperimentalFeatures.module.css'
+import { featureList } from './feature-list'
+
+const CODE = 'experimentalFeatures'
+
+/**
+ * Create an object based on elements of "featureList"
+ * */
+const populateOptions = (list) =>
+    featureList?.reduce((options, e) => {
+        options[e.name] = !isEmpty(list) && list.includes(e.name)
+        return options
+    }, {})
+
+export const ExperimentalFeatures = ({ onChange, value, ...props }) => {
+    const [options, setOptions] = useState({})
+
+    useEffect(() => {
+        setOptions(populateOptions(value[CODE]))
+    }, [value[CODE]])
+
```

Comment:
I'm not familiar with this app, but generally it would be preferred to just calculate this instead of "copying" to it own state. 

Also are eslint-rules not up to date, or not checked? 
`[value[CODE]]` should throw an eslint-warning because it's not able to be statically checked.
`populateOptions` are also missing (but doesn't matter because it's statically defined) - however it should still be needed to satisfy eslint.

```js
    const experimentalValue = value[CODE]

    const options = useMemo(() => populateOptions(experimentalValue), [experimentalValue])

    const handleCheckbox = (e) => {
        const optionsToSave = {
            ...options,
            [e.name]: e.checked,
        }
        onChange({ ...value, [CODE]: getValidKeysList(optionsToSave) })
    }
```